inference-engine/src/readers/ir_reader/ie_ir_parser.hpp

```diff
@@ -24,7 +24,9 @@
 #include <string>
 #include <vector>
 
-namespace InferenceEngine {
+#include "ie_ir_reader_namespace.hpp"
+
+namespace InferenceEngineIRReader {
```

Comment:
do we need to change namespace for all the classes / functions inside reader plugin? I suppose we need only `ie_ir_reader.cpp`, `ie_ir_reader.hpp` files, is not it?

Reply:
I think they are other colliding symbols between inference_engine_ir_reader and inference_engine_ir_v7_reader in the other files.